### DISCUSSION TITLE
Unresolved reference: FlutterFragmentActivity

### DISCUSSION THREAD

**Discussion by arnair** [flutter-stripe/flutter_stripe#938] [Q&A] [answered]
I follow the installation guide and if i do the step to change the mainactivity.kt from FlutterActivity to FlutterFragmentActivity it crash on start the app.

"MainActivity.kt: (5, 21): Unresolved reference: FlutterFragmentActivity"

Any idea?

Thanks

**Comment by jonasbark** [comment, marked as answer]
You're missing an import in your MainActivity.kt

**Reply by arnair** [reply]
omg thanks I missed the "import io.flutter.embedding.android.FlutterFragmentActivity" xD sry
